docs/docs/etl-pipeline-tutorial/7-visualize-data.md

```diff
@@ -59,13 +65,29 @@ However we can still configure our Evidence component to be dependent on multipl
     title="src/etl_tutorial/defs/dashboard/defs.yaml"
 />
 
+## 4. Execute the Evidence asset
+
+With the Evidence component configured, our assets graph should look like this.
+
+![2048 resolution](/images/tutorial/etl-tutorial/assets-evidence.png)
+
+Execute the downstream `dashboard` asset which will build our Evidence dashboards. You can now run Evidence:
```

Comment:
This step is very easy to miss -- I recommend adding the sub steps necessary to execute this asset.